tests/README.md

```diff
@@ -0,0 +1,72 @@
+## Backend development
+
+The backend test suite can be run locally. 
+
+### Start the services
+
+We need all but the main `backend` service running.  Since multiple `backend` instances are allowed, we will simply start everything up
+
+    # ENSURE THE IMAGES ARE CLEAN
+    docker-compose down
+    # SETUP ALL IMAGES
+    docker-compose up --build
+
+### Install into virtual environment
+
+Treeherder, Django, and the required libraries will ruin you local Python install. It is best to setup a virtual environment to contain the quirks it introduces.
+
+Be sure you are in the `treeherder` main directory
+
+    pip install virtualenv
+    virtualenv venv
```

Comment:
`python3 -m venv venv`. We don't need `virtualenv` anymore.

Reply:
For the record, we could also look into using `poetry`. ahal and I have been using it satisfactory in the past.